crnk-core/src/main/java/io/crnk/core/module/SimpleModule.java

```diff
@@ -1,14 +1,6 @@
 package io.crnk.core.module;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
+import com.google.common.collect.ImmutableList;
```

Comment:
this will not work, we have in crnk-core only a compile-time dependency to org.reflections:reflections, which in turn has a dependency to guava. But reflections and with that also guava is not mandatory to use crnk (to keep it small/lightweight). So application would fail without it. But Collections.immutable... should do it.

Reply:
Fixed